package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-d3-graph",
-  "version": "2.6.0",
+  "version": "2.7.0",
```

Comment:
Should the version be bumped in the PR like this or should it be bumped in a subsequent commit?